north/src/runtime/state.rs

```diff
@@ -319,6 +311,20 @@ impl State {
         }
     }
 
+    /// Stop and uninstall all the applications
+    pub async fn stop_and_uninstall_all(&mut self) -> result::Result<(), Error> {
```

Comment:
I'd rename this to `shutdown`

Reply:
That would conflict with the console shutdown function. The difference is that shutdown won't do anything if any container is running. Maybe "cleanup" or "tear_down" is a better name for this function?